content/english/python-pixel/C. Activities/b.Activity-2/b.Activity-2.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
No content for activity 1?

Reply:
Please change the path of the file to content/english/python-pixel/activities/activity-1.md (content/english/python-pixel/C. Activities/a.Activity-1/a.Activity-1.md)

Reply:
The content will be added very soon